output.c

```diff
@@ -354,29 +362,32 @@ handle_new_output(struct wl_listener *listener, void *data)
 		wlr_output_set_mode(wlr_output, preferred_mode);
 	}
 
-	server->output = calloc(1, sizeof(struct cg_output));
-	server->output->wlr_output = wlr_output;
-	server->output->server = server;
-	server->output->damage = wlr_output_damage_create(wlr_output);
-
-	server->output->mode.notify = handle_output_mode;
-	wl_signal_add(&wlr_output->events.mode, &server->output->mode);
-	server->output->transform.notify = handle_output_transform;
-	wl_signal_add(&wlr_output->events.transform, &server->output->transform);
-	server->output->destroy.notify = handle_output_destroy;
-	wl_signal_add(&wlr_output->events.destroy, &server->output->destroy);
-	server->output->damage_frame.notify = handle_output_damage_frame;
-	wl_signal_add(&server->output->damage->events.frame, &server->output->damage_frame);
-	server->output->damage_destroy.notify = handle_output_damage_destroy;
-	wl_signal_add(&server->output->damage->events.destroy, &server->output->damage_destroy);
+	struct cg_output *output = calloc(1, sizeof(struct cg_output));
```

Comment:
Please add a check here, similar to all other `calloc`s in Cage.

Reply:
Added